src/extension/manifest.json

```diff
@@ -55,7 +55,7 @@
           "sp-popover.js",
           "tab.js",
           "url-cache.js",
-          "view/json/json.html",
+          "custom-views/json/json.html",
```

Comment:
I think we should move `view-doc-source` here as well

Reply:
Agreed. Maybe just call the root folder `view` again? After all, it's not really custom (rather the default) and could be mistaken for customer-specific views...